development-updates.md

```diff
@@ -914,6 +915,11 @@ Development update from [`NC`](https://github.com/naviechan)
 
 [Update #7](https://hackmd.io/@doPpmyH4Ta-4Yc8pq3u-ZA/Hk25tFmus)
 
+
+### NC: Update 8
+
+[Update #8](https://hackmd.io/@doPpmyH4Ta-4Yc8pq3u-ZA/HyxgxaAOi)
```

Comment:
I am getting 403, can you please make the doc public?

Reply:
Oops! Fixed thanks